assets/i18n/en.json

```diff
@@ -1,4 +1,26 @@
 {
+    "I18N_ABOUT_PAGE_ABOUT_TAB_CREATE": "Create an Exploration",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_CREATE_TEXT": "about a topic you care about.",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_FEEDBACK": "Earn feedback",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_FEEDBACK_TEXT": "to improve your exploration.",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_HEADING": "About Oppia",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_SHARE": "Publish &amp; Share",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_SHARE_TEXT": "your creations with the community.",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_TEXT": "<p>Oppia's mission is to help anyone learn anything they want in an effective and enjoyable way.</p><p>By creating a set of free, high-quality, demonstrably effective lessons with the help of educators from around the world, Oppia aims to provide students with quality education &mdash; regardless of where they are or what traditional resources they have access to.</p><p>So far, educators have created more than 8,000 of these lessons, which we call <em>explorations</em>. And they serve almost 250,000 students worldwide.</p><p>Explorations help students learn in a fun and creative way, using videos, images, and open-ended questions. And since students often have similar misconceptions, Oppia also provides educators the ability to address these misconceptions directly within explorations, empowering them to provide targeted feedback to multiple students at a time.</p><p>If you're a student interested in learning with Oppia, you can <a href=\"/library\">begin your learning adventure by browsing our explorations</a>.</p><p>If you're a teacher interested in impacting the lives of students around the world, you can apply to join our <a href=\"/teach\">Teach with Oppia</a> program, aimed at providing lessons for topics that students typically find difficult.</p><p>By teaching with Oppia, you can improve your skills in communication and empathy while helping to improve education for students around the world.  Or, if you're not ready to teach yet, you can still share feedback on lessons to help make them better for other students!</p><p>Whether you're a K-12 educator, a graduate student, or an individual who's passionate about a specific subject and wants to share your knowledge, Oppia welcomes you. Join the community and start exploring with us.</p>",
```

Comment:
Hi @Nikerabbit @siebrand @nemobis -- just wanted to check a quick thing: is this text OK for a translation entry, or is it too long? If the latter, do you have any advice on what the best thing to do would be?

Thanks!